debian/control.workstation

```diff
@@ -0,0 +1,9 @@
+Package: securedrop-workstation-grsec
+Section: admin
+Architecture: ${DEBARCH}
+Pre-Depends: qubes-kernel-vm-support (>=4.0.31)
```

Comment:
I assume we can drop this now? The 4.1 bookworm VM I have is on `4.1.19+deb12u1` already.

Reply:
Per https://github.com/freedomofpress/securedrop-builder/commit/de34206a8a92690333c7defafa4752b4717b2c8c this was also only needed for the dkms build, so we can drop it too.

Reply:
This analysis didn't account for the switch to minimal templates, which are missing this package. So we do need a dependency on qubes-kernel-vm-support, though we can make it unversioned.